lnrpc/routerrpc/router.proto

```diff
@@ -604,17 +604,29 @@ message ForwardHtlcInterceptRequest {
     */
     CircuitKey incoming_circuit_key = 1;
 
+    // The incoming htlc amount.
+    uint64 incoming_amount_msat = 5;
+
+    // The incoming htlc expiry.
+    uint32 incoming_expiry = 6;
+
     /*
     The htlc payment hash. This value is not guaranteed to be unique per
     request.
     */
-    bytes htlc_payment_hash = 2;
+    bytes payment_hash = 2;
+
+    // The outgoing channel id for this forwarded htlc.
```

Comment:
docs here should note that this may not be the actual channel this gets forwarded over due to non-strict forwarding. could consider making this `requested_outgoing_chan_id` as well

Reply:
Added